### PR TITLE
Header style

### PR DESCRIPTION
**Pull request by Chebi97**

### CODE REVIEW

**Comment by andresk727** [comment]
slider comments
-Make text and icons bigger
-Align name and surname when it doesn't fit

**Comment by andresk727** [comment]
header comments 1
header comments 2
Unify header style. In the Gather header the name is lower case and the profile image is at the left of the name, while in the others the name is in upper case and the profile image at the right.

File: src/components/common/Drawer.js [hunk 1]
```diff
@@ -22,57 +32,88 @@ class Drawer extends Component {
...
+  logout: PropTypes.func.isRequired,
+  changeRole: PropTypes.func.isRequired,
+  navigator: PropTypes.object.isRequired,
+};
 
-Drawer.defaultProps = {};
+Drawer.defaultProps = {
+  user: 'not assigned',
+  role: 'not assigned',
+};
```
Comment (andresk727) [inline src/components/common/Drawer.js]: Did you use 'not assigned' for any special reason? Is better to use false instead.

File: src/Application.js [hunk 2]
```diff
@@ -19,10 +19,10 @@ class Application {
         right: {
           screen: Screens.Drawer,
           passProps: {},
-          disableOpenGesture: false,
-          fixedWidth: '20%',
+          disableOpenGesture: true,
+          fixedWidth: 600,
         },
-        disableOpenGesture: false,
+        disableOpenGesture: true,
```
Comment (dsusviela) [inline src/Application.js]: There's an issue with our version of react-native-navigation that is making this option useless. Its out of your control for now.

File: src/components/common/Drawer.js [hunk 3]
```diff
@@ -22,57 +32,94 @@ class Drawer extends Component {
...
+                  onPress={this.changeRole}
+                />
+              ),
+              Bale: (
+                <CustomButton
+                  icon={BaleIcon}
+                  title="bale button"
+                  textStyle={TextStyles.drawerButtons}
+                  style={{ width: 140 }}
+                  onPress={this.changeRole}
```
Comment (dsusviela) [inline src/components/common/Drawer.js]: I dont think buttons should be doing this.

**Pull request merged**